crates/adapters/src/server/mod.rs

```diff
@@ -835,6 +838,28 @@ async fn metrics_handler(
     }
 }
 
+/// Retrieve circuit graphs.
```

Comment:
This is a misleading name, since we will probably use "graphs" to refer to the dataflow graph too. Maybe "Statistics"?